packages/build/bin/build-css.mjs

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
```

Comment:
`git mv` didn't work it looks like

Reply:
Yeah, doesn't look very helpful.  Highlights are:


Can't import json now. Reading with this lib
```
import readPkgUp from 'read-pkg-up'

const { packageJson } = readPkgUp.sync()
```

Can't require files any more. Using import.

```
  const { default: defaultExport } = await import(inputFilePath)
  js = defaultExport
```

And because of the import() usage, wrapped in `async main()`.